README.md

```diff
@@ -59,6 +59,7 @@ KubeEdge consists of cloud part and edge part.
 | KubeEdge 1.13 | +               | ✓               | ✓               | ✓               | -               | -               | -               | -               |
 | KubeEdge 1.14 | +               | +               | ✓               | ✓               | ✓               | -               | -               | -               |
 | KubeEdge 1.15 | +               | +               | +               | +               | ✓               | ✓               | ✓               | -               |
+| KubeEdge 1.16 | +               | +               | +               | +               | +               | ✓               | ✓               | ✓               |
```

Comment:
The same modification has been proposed in PR #5412:  https://github.com/kubeedge/kubeedge/pull/5412/commits/c2bbc5ae6db6a5f3d19e82e73605a03ae3356fed